### PR TITLE
Bug 1185808 - "Output to" cannot display well in buildconfig description

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by smarterclayton** [comment]
Just reassign it back to me - someone else can grab.  Should be a few minutes to fix.

----- Original Message -----

**Comment by smarterclayton** [comment]
Doh, I thought this was a bugzilla response.

File: pkg/cmd/cli/describe/describer.go [hunk 1]
```diff
@@ -92,7 +92,14 @@ func (d *BuildDescriber) DescribeParameters(p buildapi.BuildParameters, out *tab
 			formatString(out, "Ref", p.Source.Git.Ref)
 		}
 	}
-	formatString(out, "Output To", p.Output.To)
+	if p.Output.To != nil {
+		if p.Output.To.Namespace != "" {
+			formatString(out, "Output to", p.Output.To.Namespace+":"+p.Output.To.Name)
```
Comment (smarterclayton) [inline pkg/cmd/cli/describe/describer.go]: fmt.Sprintf instead of string concat, use slash to denote namespace.

**Comment by bparees** [comment]
updated.

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**